src/site/topics/inheritance/inheritance.rst

```diff
@@ -40,113 +44,137 @@ Terminology
     * An ``Object`` is also a superclass of ``Bag``, ``SortedBag``, and ``IndexedBag``
 
 
+
 Object
 ======
 
-* The class ``Object`` is the root of the hierarchy
-* Every class you make will inherit from the ``Object`` class
-    * This is done automatically, so you do not ``extend`` it explicitly
+* The ``Object`` class is always at the root of the hierarchy
+* Every class in Java will inherit from the ``Object`` class
 
-* Since we inherit everything from the superclasses we inherit from, this means that we get all methods from the ``Object`` class without having to write them ourselves
-    * Although, we almost always override them if were using them
-* We have seen and used a few of these already
-    * ``equals``
-    * ``toString``
+    * This is done automatically; it is not necessary to ``extend`` ``Object`` explicitly
 
-* However, we ended up *overriding* these methods a bunch of times
-    * We inherited them, but we wrote our own implementations for our needs
 
-* The inherited behaviour of ``equals`` checks if the references are to the same exact object
-    * ``x.equals(y)`` checks if ``x`` and ``y`` are the same object (same as ``x == y``)
+* Child classes inherit everything from their superclasses
+* This means that all the methods from ``Object`` are inherited for free
+
+    * ``toString``
+    * ``equals``
+    * ``hashCode``
+    * Although,
 
-* The inherited behaviour of ``toString`` is to return a string of the object's class name + @ + the hash code of the object
 
-* If we do not override these methods in our classes, we can still use them, but get the default behaviour
+* However, they are typically overridden with class specific functionality if they are to be used
```

Comment:
"although" looks out of place here. the next bullet already starts with "however"